asn1/InterledgerProtocol.asn

```diff
@@ -10,12 +10,9 @@ IMPORTS
     Address,
     Timestamp
     FROM InterledgerTypes
-
-    MetadataEntry
-    FROM MetadataEntry
 ;
 
-InterledgerProtocolPayment ::= SEQUENCE {
+InterledgerProtocolPaymentMessage ::= SEQUENCE {
```

Comment:
We should probably call this `InterledgerProtocolPaymentPacket`, because it's in the memo of a transfer, not sent as a message.